src/main/resources/assets/thaumictinkerer/lang/ga_ie.lang

```diff
@@ -0,0 +1,138 @@
+itemGroup.thaumictinkerer=Thaumic Tinkerer
+
+
+item.thaumictinkerer:share_book.name=Tome of Roinnt Eolais
+item.thaumictinkerer:share_book.0=Dialann Éisc Óir
+
+
+item.thaumictinkerer:black_quartz.name=Grianchloch dheatach
+item.thaumictinkerer:connector.name=Ceanglóir Transvector
+
+item.thaumictinkerer:energetic_nitor.name=Nitor fuinniúil
+item.thaumictinkerer:form_revealer.name=Multiblock treoirleabhar
+item.thaumictinkerer:spellbinding_cloth.name=Éadach Litrithe
+item.thaumictinkerer:soul_mould.name=Soul Mould
+item.thaumictinkerer:blood_sword.name=Claíomh Spiorad Cursed
+item.thaumictinkerer:mob_aspect.name=Mob Aspect
+
+tile.thaumictinkerer:funnel.name=Essentia Funnel
+tile.thaumictinkerer:funnel.0=Fiú Aistriú
+
+tile.thaumictinkerer:dissimulation.name=Bloc Díshamhlaithe
+
+tile.thaumictinkerer:transvector_interface.name=Comhéadan Transvector
+tile.thaumictinkerer:transvector_dislocator.name=Dislocator Transvector
+
+tile.thaumictinkerer:repairer.name=Thaumic Restorer
+tile.thaumictinkerer:animation_tablet.name=Dynamism Tablet
+tile.thaumictinkerer:magnet.name=Mealltóir Cinéiteach
+tile.thaumictinkerer:mob_magnet.name=Corporeal Attractor
+tile.thaumictinkerer:osmotic_enchanter.name=Osmotic Enchanter
+tile.thaumictinkerer:enchantment_pillar.name=Enchanting Colún
+tile.thaumictinkerer:black_quartz_block.name=Bloc Grianchloch Dubh
+tile.thaumictinkerer:example.name=Bloc Áititheoir
+
+thaumictinkerer.TELEKENESIS.name=Telekenesis
+thaumictinkerer.mobaspect.type=Aspect: %s
+thaumictinkerer.mobaspect.invalid=Gné Neamhbhailí
+
+
+thaumictinkerer.config.title=Thaumic Tinkerer Cumraigh
+thaumictinkerer.config.talisman=Talisman of Remedium Uses
```

Comment:
Should uses be translated too?

Reply:
This will be needed to be further looked into as translation progresses. At this early stage, it lead to a clunky sentence structure.